### ISSUE TITLE
A11y issue: Spinner

### ISSUE THREAD

**Issue by joasmi723** [cengage/react-magma#272]
The Spinner component is throwing an a11y error, because it uses `aria-label` on the span that displays the spinner.  Under 2.1 guidelines, only interactive elements should have aria-labels.  Instead we should use the `VisuallyHidden` component.
